SETUP/manual_web/split_test/h_split.js

```diff
@@ -0,0 +1,12 @@
+/* global $ splitControl */
+$(function () {
+    let reDraw = $.Callbacks();
+
+    $(window).resize(function () {
```

Comment:
That could be a good approach. It seems possible that there could be other ways (apart from window.resize() that the container could get resized so the splitter would break but I haven't found any.
It would be best to detect the container resizing directly by ResizeObserver. This would also remove the need for the resize return parameter but as Casey observed in https://github.com/DistributedProofreaders/dproofreaders/pull/298 it is not supported in all the browsers we want to support. I tried it and it works well in Firefox but only partially in Edge and Chrome: it doesn't get triggered by a resize induced by javascript. Another way to do it I found on stackoverflow is to continuously check the size with a timer function but I didn't want to do that.

Reply:
This is now default and moved into config